src/js/components/NewsFeedPost.jsx

```diff
@@ -123,45 +124,31 @@ export default class NewsFeedPost extends React.Component {
       );
     }
 
-
-    let image = null;
-    if (media_link && media_link !== '') {
-      image = (
-        <span className="show-image link clickable no-selection" onClick={this.showImage}>
-          <img src={media_link}/>
-        </span>
-      );
-    }
-
     return (
-      <div className="newsfeed-post">
-        <div className="profile-photo inline">
-          <img alt="poster" src={poster_icon} className="user-photo no-selection" />
-        </div>
-        <div className="post-text inline">
-          <div className="right-options">
```

Comment:
I'm not saying that this styling should be in here but I didn't know where to comment that the associated "flag" button no longer seems to exist....

Marked under the actions variable
```
    let actions = null;
    if (owner) {
      actions = (
        <FaTrashO
          className="clickable right-icons no-selection"
          onClick={this.deletePost}
          size={20}
        />
      );
    } else {
      actions = (
        <FaFlag
          className="clickable right-icons no-selection"
          onClick={this.changeFlag}
          style={flagColor}
          size={20}
        />
      );
    }
...............
          <div className="right-options">
            {actions}
          </div>
```